c_tests/tests/simple.c

```diff
@@ -39,7 +38,11 @@
 //     jit-state: exit-jit-code
 //     jit-state: enter-jit-code
 //     i=1
-//     simple: guard-failure
+//     Stopgap
+//     ...
+//     Indirect: 10006 ...
```

Comment:
What are the numbers being matched here again?

Reply:
These are some live values we expect. When the guard fails we have just done `res += 2` within a loop, so we are expecting at least two live variables, before the addition (`10006`) and after the addition (`10008`). There are other live variables but these are an easy way to check that stackmaps are working.